include/librealsense2/dds/topics/device-info/deviceInfo.idl

```diff
@@ -0,0 +1,25 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+module librealsense
+{
+    module dds
+    {
+        module topics
+        {
+            module raw
+            {
+                const string DEVICE_INFO_TOPIC_NAME = "realsense/device-info";
```

Comment:
Is this normal, to put it outside the class? Can it go inside? `device_info::TOPIC_NAME`?

Reply:
I will try and update

Reply:
I did some testings and the `fastddsgen` does not allow `const` specifier inside a message